nixos/tests/prometheus-exporters.nix

```
@@ -1677,7 +1710,7 @@ mapAttrs
       testScript = ''
         ${nodeName}.start()
         ${concatStringsSep "\n" (map (line:
-          if (builtins.substring 0 1 line == " " || builtins.substring 0 1 line == ")")
+          if (line == "" || builtins.substring 0 1 line == "#"|| builtins.substring 0 1 line == " " || builtins.substring 0 1 line == ")")
```

This is starting to get long and diff unfriendly, let's give it some more space. I also extended the definition of empty line a bit, not sure if it's a good or bad change.

```suggestion
          if builtins.any (b: b) [
            (builtins.match "^[[:space:]]*$" line)
            (builtins.substring 0 1 line == "#")
            (builtins.substring 0 1 line == " ")
            (builtins.substring 0 1 line == ")")
          ]
```

---

When I try that, I get this confusing error:
```
nix-build -A nixosTests.prometheus-exporters.restic
error:
       … while evaluating the attribute 'drvPath'

         at /home/sebastian/Repos/nixpkgs/lib/customisation.nix:260:7:

          259|     in commonAttrs // {
          260|       drvPath = assert condition; drv.drvPath;
             |       ^
          261|       outPath = assert condition; drv.outPath;

       … while calling the 'derivationStrict' builtin

         at /builtin/derivation.nix:9:12: (source not available)

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: value is null while a Boolean was expected
```

---

Nevermind `builtins.match` returns either null or a List, not a boolean.
After realising the fix is easy.